cpp/valgrind.supp

```diff
@@ -21,4 +21,33 @@
     Memcheck:Cond
     fun:*CastFunctor*BooleanType*
 }
-
+{
+    <re2>:Conditional jump or move depends on uninitialised value(s)
+    Memcheck:Cond
+    ...
+    fun:_ZN3re23RE2C1E*
+}
+{
+    <re2>:Use of uninitialised value of size 8
+    Memcheck:Value8
+    ...
+    fun:_ZN3re23RE2C1E*
+}
+{
+   <static_exported_context_functions>
+   Memcheck:Leak
+   ...
+   fun:_GLOBAL__sub_I_context_helper.cc
```

Comment:
Hmm... Before we add suppressions for these, perhaps it would be possible to instead reclaim the memory automatically? (for example by using `unique_ptr` or `shared_ptr` instead of raw pointers).